### PR TITLE
Update _index.md

### PR DESCRIPTION
**Pull request by erwinpalma**
## Related Issue

- Fixes <issue_link>

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Comment by erwinpalma** [comment]
## Adding users from the commad line.

  The fastest way to create a user is by executing the following command:

 
 `     ./ocis accounts add --username johndoe --mail jondoe@none.com --password p123p123 --description  "John Doe's account" --displayname "John Doe" --enabled 
`
  For detailed information just type:

 
```
      ./ocis accounts add --help
     NAME:
        ocis accounts add - Create a new account

      USAGE:
        ocis accounts add [command options] [arguments...]

      OPTIONS:
        --grpc-namespace value                Set the base namespace for the grpc namespace (default: "com.owncloud.api") [$ACCOUNTS_GRPC_NAMESPACE]
        --name value                          service name (default: "accounts") [$ACCOUNTS_NAME]
        --enabled                             Enable the account (default: false)
        --displayname value                   Set the displayname for the account
        --username value                      Username will be written to preferred-name and on_premises_sam_account_name
        --preferred-name value                Set the preferred-name for the account
        --on-premises-sam-account-name value  Set the on-premises-sam-account-name
        --uidnumber value                     Set the uidnumber for the account (default: 0)
        --gidnumber value                     Set the gidnumber for the account (default: 0)
        --mail value                          Set the mail for the account
        --description value                   Set the description for the account
        --password value                      Set the password for the account
        --password-policies value             Possible policies: DisableStrongPassword, DisablePasswordExpiration
        --force-password-change               Force password change on next sign-in (default: false)
        --force-password-change-mfa           Force password change on next sign-in with mfa (default: false)
        --help                                Show the help (default: false)
```

## How to delete users.
First of all, get the user id, to know it run the following command.

```
    ./ocis accounts list | grep "John"
    | dd8bc84c-ff57-4c72-9db7-e6ff435acfd8 | John Doe | jondoe@none.com  | true |
```


The ID is "dd8bc84c-ff57-4c72-9db7-e6ff435acfd8", so to proced to delete it just run the command.
`
    ./ocis accounts rm dd8bc84c-ff57-4c72-9db7-e6ff435acfd8
`

File: docs/extensions/accounts/_index.md [hunk 1]
```diff
@@ -17,4 +17,42 @@ oCIS needs to be able to identify users. Without a non reassignable and persiste
...
+
+## Adding users from the commad line.
+
+The fastest way to create a user is by executing the following command:
+
+
+    ./ocis accounts add --username johndoe --mail jondoe@none.com --password p123p123 --description  "John Doe's account" --displayname "John Doe" --enabled 
+
+For detailed information just type:
+
```
Comment (dragotin) [inline docs/extensions/accounts/_index.md:31]: I would vote against printing the whole help in the docs. It is subject to change.

I'd say it is enough to say:

    For detailed information just type òcis accounts add --help`. 

Also keeps the docs compact.

**Review by dragotin — CHANGES_REQUESTED** [review]

**Comment by dragotin** [comment]
Thanks for your PR, @erwinpalma 

If you work on the PR again, could you please also fix the typo "proced" -> "proceed" in the How-to-delete paragraph? Thanks!

**Comment by butonic** [comment]
we are planning to move user management out of ocis, with that step docs for it will become outdated, which is why I am closing this. Take a look at https://github.com/libregraph/idm which we plan to integrate instead of building our own identity management.

**Pull request closed without merging**